app/src/main/java/com/nervousfish/nervousfish/modules/pairing/ZXingQRAdapter.java

```diff
@@ -0,0 +1,29 @@
+package com.nervousfish.nervousfish.modules.pairing;
+
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+import com.nervousfish.nervousfish.service_locator.IServiceLocatorCreator;
+import com.nervousfish.nervousfish.service_locator.ModuleWrapper;
+
+/**
+ * An adapter to the ZXing QR code library
+ */
+public final class ZXingQRAdapter implements IQRHandler {
```

Comment:
Are you already sure we should use ZXing? I think this class also should not be here yet because we are not going to implement it at this moment